lib/helm-util.js

```diff
@@ -66,6 +66,7 @@ exports.walkSync = function (dir, filelist, fileToFind) {
     });
     return filelist;
 };
+exports.walkSync = walkSync;
 function downloadHelm(version) {
```

Comment:
is it possible to just use `export function` instead of having a separate line for setting the module exports?

Reply:
@davidgamero looks like it's not.
I checked docs for tsconfig compiler and there is no option for that :
https://www.typescriptlang.org/tsconfig

Reply:
okay gotcha